src/ambassadors/core.ts

```diff
@@ -558,6 +634,10 @@ const ambassadors = {
       "Snork was part of an educational program at a zoo and was rehomed to Alveus.",
     mission:
       "She is an ambassador for the exploitation of wildlife in the fur trade.",
+    native: {
+      text: "Andes Mountains (South America)",
```

Comment:
Should this be marked as domesticated?

Reply:
yes, thank you

their entry states 

> Domesticated chinchillas are thought to be of the C. lanigera

of which Snork and Moomin both are